workshops/models.py

```diff
@@ -1850,11 +1850,11 @@ def build_choice_field_with_other_option(choices, default, verbose_name=None):
 
 @reversion.register
 class TrainingRequest(ActiveMixin, CreatedUpdatedMixin, models.Model):
-    STATES = (
+    STATES = [
         ('p', 'Pending'),  # initial state
         ('a', 'Accepted'),  # state after matching a Person record
         ('d', 'Discarded'),
-    )
+    ]
```

Comment:
Because [here](https://github.com/swcarpentry/amy/pull/1030/files#diff-50e7b27c35104059c102375ddba0f88fR393) it's more readable to write:

    [('', 'Pending or accepted')] + TrainingRequest.STATES

rather than

    (('', 'Pending or accepted'),) + TrainingRequest.STATES